migrate/v0_13/migrate.go

```diff
@@ -78,6 +89,211 @@ func MigrateCDP(oldGenState v0_11cdp.GenesisState) v0_13cdp.GenesisState {
+		hardSupplyGenAccumulationTimes = append(hardSupplyGenAccumulationTimes, gat)
+	}
+	hardBorrowGenAccumulationTimes := v0_13incentive.GenesisAccumulationTimes{}
+	for _, rp := range params.HardBorrowRewardPeriods {
+		gat := v0_13incentive.NewGenesisAccumulationTime(rp.CollateralType, GenesisTime, sdk.ZeroDec())
+		hardBorrowGenAccumulationTimes = append(hardBorrowGenAccumulationTimes, gat)
+	}
+
+	hardDelegatorGenAccumulationTimes := v0_13incentive.GenesisAccumulationTimes{}
+
+	for _, rp := range params.HardDelegatorRewardPeriods {
+		gat := v0_13incentive.NewGenesisAccumulationTime(rp.CollateralType, GenesisTime, sdk.ZeroDec())
+		hardDelegatorGenAccumulationTimes = append(hardDelegatorGenAccumulationTimes, gat)
+	}
+
+	usdxClaims := v0_13incentive.USDXMintingClaims{}
+	usdxClaimMap := make(map[string]v0_13incentive.USDXMintingClaim)
+	claimEndMap := make(map[uint64]time.Time)
+
+	for _, cp := range incentiveGS.ClaimPeriods {
+		claimEndMap[cp.ID] = cp.End
+	}
+
+	for _, claim := range incentiveGS.Claims {
+		if claimEndMap[claim.ClaimPeriodID].Before(GenesisTime) {
+			continue
+		}
+		newClaim, ok := usdxClaimMap[claim.Owner.String()]
+		if !ok {
+			newClaim = v0_13incentive.NewUSDXMintingClaim(claim.Owner, claim.Reward, v0_13incentive.RewardIndexes{v0_13incentive.NewRewardIndex(claim.CollateralType, sdk.ZeroDec())})
+		} else {
+			newClaim.Reward = newClaim.Reward.Add(claim.Reward)
+			_, found := newClaim.RewardIndexes.GetRewardIndex(claim.CollateralType)
+			if !found {
+				newClaim.RewardIndexes = append(newClaim.RewardIndexes, v0_13incentive.NewRewardIndex(claim.CollateralType, sdk.ZeroDec()))
+			}
+		}
+		usdxClaimMap[newClaim.Owner.String()] = newClaim
+	}
+
+	for _, claim := range usdxClaimMap {
+		usdxClaims = append(usdxClaims, claim)
+	}
+
+	hardClaims := v0_13incentive.HardLiquidityProviderClaims{}
+	hardClaimMap := make(map[string]v0_13incentive.HardLiquidityProviderClaim)
+
+	for _, claim := range hardGS.Claims {
+		newClaim, ok := hardClaimMap[claim.Owner.String()]
+		if !ok {
+			// if claim.Type == "lp" -- hard supply
+			// if claim.Type == "stake" -- hard delegator
+			// hard barrow always empty
+			delegatorIndexes := v0_13incentive.RewardIndexes{}
+			supplyIndexes := v0_13incentive.MultiRewardIndexes{}
+			borrowIndexes := v0_13incentive.MultiRewardIndexes{}
+			if claim.Type == v0_11hard.Stake {
+				delegatorIndexes = v0_13incentive.RewardIndexes{v0_13incentive.NewRewardIndex(claim.DepositDenom, sdk.ZeroDec())}
+			}
+			if claim.Type == v0_11hard.LP {
+				supplyIndexes = v0_13incentive.MultiRewardIndexes{v0_13incentive.NewMultiRewardIndex(claim.DepositDenom, v0_13incentive.RewardIndexes{v0_13incentive.NewRewardIndex("hard", sdk.ZeroDec())})}
+			}
+			newClaim = v0_13incentive.NewHardLiquidityProviderClaim(claim.Owner, sdk.NewCoins(claim.Amount), supplyIndexes, borrowIndexes, delegatorIndexes)
+		} else {
+			newClaim.Reward = sdk.NewCoins(claim.Amount)
```

Comment:
```suggestion
			newClaim.Reward = newClaim.Reward.Add(claim.Amount)
```
If we're combining multiple claims from a single owner into an aggregated new claim then I think we need to increment the new claim's reward here instead of overwriting it